src/Symfony/Component/Form/Extension/Core/Type/FormType.php

```diff
@@ -207,8 +213,10 @@ public function configureOptions(OptionsResolver $resolver)
             'action' => '',
             'attr' => $defaultAttr,
             'post_max_size_message' => 'The uploaded file was too large. Please try to upload a smaller file.',
+            'upload_max_size_message' => $uploadMaxSizeMessage,
```

Comment:
You could add `// internal` at the end of the line, and/or maybe add a comment somewhere explaining why you define it as a callable.